src/java/org/apache/cassandra/tools/nodetool/Assassinate.java

```diff
@@ -17,27 +17,25 @@
  */
 package org.apache.cassandra.tools.nodetool;
 
-import static org.apache.commons.lang3.StringUtils.EMPTY;
-import io.airlift.airline.Arguments;
-import io.airlift.airline.Command;
-
 import java.net.UnknownHostException;
 
 import org.apache.cassandra.tools.NodeProbe;
-import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
+import picocli.CommandLine;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
 
-@Command(name = "assassinate", description = "Forcefully remove a dead node without re-replicating any data.  Use as a last resort if you cannot removenode")
-public class Assassinate extends NodeToolCmd
+@CommandLine.Command(name = "assassinate", description = "Forcefully remove a dead node without re-replicating any data.  Use as a last resort if you cannot removenode")
+public class Assassinate extends AbstractCommand
 {
-    @Arguments(title = "ip address", usage = "<ip_address>", description = "IP address of the endpoint to assassinate", required = true)
-    private String endpoint = EMPTY;
+    @CommandLine.Parameters(description = "IP address of the endpoint to assassinate", arity = "1")
+    public String ip_address = EMPTY;
```

Comment:
Does the help text rely on the variable name for `ip_address`? If so, I'm not sure that's optimal, as someone might come along and try to rename this field...

Reply:
Fixed.